src/wled2.ts

```diff
@@ -99,6 +99,8 @@ export = (RED: Red): void => {
       bri: payload?.brightness ?? Number(this.config.brightness),
     } as IWledState;
 
+    const effect = Number(await getEffectFromString(this.config.address, payload?.effect, this.config.effect));

```

Comment:
This will result in the query to the WLED device to get the list of effects happening every time the node is triggered. The list of effects changes rarely, like on the level of once every few months, so calling every time the node is triggered seems overkill.

Is there a way to make a single call at node startup time to get the list of effects and then store those for use when the node is actually triggered?